src/app/beer_garden/plugin.py

```diff
@@ -203,15 +206,17 @@ def update(
     updates = {}
 
     if new_status:
-        updates["set__status"] = new_status
-        updates["set__status_info__heartbeat"] = datetime.utcnow()
+        updates["set__instances__S__status"] = new_status
+        updates["set__instances__S__status_info__heartbeat"] = datetime.utcnow()
```

Comment:
It wouldn't let me put a comment above, but I thought he DB query for instances by ID would no longer work. (line 198)

Reply:
Arg, I missed one. Good catch, thank you!

Reply:
Ok, this should be resolved